src/test/java/ru/yandex/practicum/filmorate/model/FilmTest.java

```diff
@@ -0,0 +1,145 @@
+package ru.yandex.practicum.filmorate.model;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import ru.yandex.practicum.filmorate.ItemForTest;
+import ru.yandex.practicum.filmorate.controller.validation.FixedValues;
+
+import javax.validation.ConstraintViolation;
+import javax.validation.Validation;
+import javax.validation.Validator;
+import javax.validation.ValidatorFactory;
+
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class FilmTest {
+    private static Validator validator;
+    private static Film film;
+
+    @BeforeAll
+    static void setupValidation() {
+        ValidatorFactory factory = Validation.buildDefaultValidatorFactory();
+        validator = factory.getValidator();
+    }
+
+    @BeforeEach
+    void setupFilm() {
+        film = ItemForTest.setDefaultTestFilm(1);
+    }
+
+    @Test
+    void filmIsValid() {
+
+        Set<ConstraintViolation<Film>> constraintViolations = validator.validate( film );
+
+        assertEquals( 0, constraintViolations.size() );
+    }
+
+    @Test
+    void nameAndDescriptionNull() {
+        film.setName(null);
+        film.setDescription(null);
+
+        Set<ConstraintViolation<Film>> constraintViolations = validator.validate( film );
+        List<String> violationMessages = constraintViolations.stream().map(v -> v.getMessage())
```

Comment:
Стримы лучше бить на несколько строк, так их удобнее читать:
```suggestion
        List<String> violationMessages =
            constraintViolations.stream()
                .map(v -> v.getMessage())
                .collect(Collectors.toList());
```

Reply:
исправлено